TLSharp.Core/TelegramClient.cs

```diff
@@ -42,9 +45,29 @@ public TelegramClient(int apiId, string apiHash, ISessionStore store = null, str
             TLContext.Init();
             _apiHash = apiHash;
             _apiId = apiId;
+            _webProxy = webProxy;
 
             _session = Session.TryLoadOrCreateNew(store, sessionUserId);
-            _transport = new TcpTransport(_session.ServerAddress, _session.Port);
+            _transport = new TcpTransport(_session.ServerAddress, _session.Port, _webProxy);
+        }
+
+        static public WebProxy CreateWebProxy(string httpProxyHost, int httpProxyPort = 80,
```

Comment:
It was to provide a sample of code to initialize a WebProxy from the parameters but I removed it